lessons/Early-Math/evaluating-exponents/langs/en-us/index.adoc

```diff
@@ -119,49 +139,61 @@ Ensure that students are comfortable shifting between representations before dis
 |@math{5^2}							| @math{5 \times 2}
 |@show{(coe  '(expt 5 2))}		| @show{(coe  '(* 5 2))}
 |===
+
 }
 
+@teacher{
 With a basic understanding of how exponents work, students are ready to discover how Circles of Evaluation can help them in analyzing more complicated expressions with exponents.
+}
 
 === Investigate
 
 @QandA{
-Look at the two Circles of Evaluation below:
```

Comment:
In general, I find it much more readable to put the circles first and _then_ asking questions about them. I'm going to move these tables so they come before the questions - let me know if you think this makes it easier to read?

Reply:
works for me. (Don't really have a strong preference one way or the other!)